src/model/boss.ts

```diff
@@ -1,10 +1,11 @@
 export interface Boss {
-    numberOfBosses: number
-    phased: boolean
-    skippablePhases: boolean
-    introPhase: boolean
-    spawned: boolean
-    bossArena: boolean
-    names: string[]
-    notes: any
+  numberOfBosses: number
+  phased: boolean
+  skippablePhases: boolean
+  introPhase: boolean
+  spawned: boolean
+  bossArena: boolean
+  names: string[]
+  notes: any
+  [propName: string]: number | boolean | string | string[]
```

Comment:
Due to this Addition we get an error in the atlasNode store, where we convert the Values from the Json to AtlasNode Objects.

Reply:
Yeah I figured this would cause an issue elsewhere, having trouble getting my generic component to play nice with typescript hahahaha